### DISCUSSION TITLE
I downloaded the models locally from HF, how can I load the model?

### DISCUSSION THREAD

**Discussion by bitcoinvsalts** [haotian-liu/LLaVA#1163] [General]
I need to download the LLava model ([https://huggingface.co/liuhaotian/llava-v1.5-13b/tree/main](https://huggingface.co/liuhaotian/llava-v1.5-13b/tree/main)) on my local machine and be able to load the model from those files. How is it possible?

The following code will re-download all the files every time even when the models are already cached:

```
         # Llava model
        logger.info(f"lava loading")
        model_name = get_model_name_from_path('liuhaotian/llava-v1.5-7b')
        tokenizer, model, image_processor, context_len = load_pretrained_model('liuhaotian/llava-v1.5-7b', None, model_name, False, True, device='cuda')

```

`[Agent:v0.3.3 i-0dc682] diffusion_pytorch_model.fp16.safetensors: 99%|█████████▉| 5.11G/5.14G [00:37<00:00, 131MB/s][A`

Thanks.
HF

**Comment by Pompey21** [comment]
In what format would you like to download the model?
